conf/kvm.conf

```diff
@@ -29,11 +29,6 @@ ip = 192.168.122.105
 # Example (Snapshot1 is the snapshot name):
 # snapshot = Snapshot1
 
-# (Optional) Specify the name of the network interface that should be used
-# when dumping network traffic from this machine with tcpdump.
-# Example (virbr0 is the interface name):
-# interface = virbr0
```

Comment:
Correct. This is a port of a fix that I used in `cuckoo-modified` to make sniffing work with the way I'm routing Tor. In `cuckoo-modified` I had a check to honor the per-VM interfaces. However, when I tried that check upstream, that config option always got applied and overwrote after my code tried to set the interface to be sniffed. Rather than changing more of the scheduling, I figured it was easier/safer to just remove the option. The only reason I kept the KVM global interface setting is to have a fallback, and prevent Cuckoo from scream about a missing config option on startup.